__tests__/app.test.js

```diff
@@ -199,3 +198,40 @@ describe("PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("200: responds with a reviews array", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        console.log(reviews);
```

Comment:
just because I can see you've removed one too, I'd say remember to remove console.logs form your code before you commit and push. If it doesnt need to be in there for production or for other devs, it shouldn't get added to git hub